src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -19,9 +19,18 @@ import ElementWrapper from "./ElementWrapper";
 
 const { reportError } = errorReporting;
 
-const getInputErrors = (rawErrors: string[]): string[] =>
+const getInputErrors = (rawErrors: string[]): string[] => {
   // Don't display empty input errors as these are too visually noisy
-  rawErrors?.filter((err) => !err.startsWith("should match format")) ?? [];
+  const errorMsgs = [
+    "should match format",
+    "should NOT be shorter than 1 characters",
+  ];
+  return (
+    rawErrors?.filter((err) => !errorMsgs.some((msg) => err.startsWith(msg))) ??
+    []
+  );
```

Comment:
I was wondering how you felt about RJSF's pattern for handling custom error messages. https://react-jsonschema-form.readthedocs.io/en/latest/usage/validation/#custom-error-messages. 
It seems to case on the validators as opposed to the error messages they output.